src/SDKs/ManagedServices/ManagedServices.Test/ManagedServices.Test.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.test.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  <PropertyGroup>
+    <PackageId>Foo</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <AssemblyName>Foo</AssemblyName>
+    <Description>BatchAI.Tests;</Description>
```

Comment:
Please create an `AssemblyInfo.cs` file for the test project too and ensure the properties are appropriately set

Reply:
ok will do